option-descriptions.js

```diff
@@ -0,0 +1,29 @@
+module.exports = [
+  {
+    header: 'Check peer deps',
+    content: 'Verifies that the peerDependency requirements of all top level dependencies are satisfied'
+  }, {
+    header: 'Options',
+    optionList: [
```

Comment:
This information is essentially a duplicate of [here](https://github.com/Arcanemagus/check-peer-deps/pull/12/files#diff-101dd0c1b8777af412dc01f71722ded4R177), these should just be merged together into the main file.